Update gtfs.markdown

Friedrieck opened the pull request:
To clarify what is expected for the offset parameter (based on the error message I got in the log).

## Proposed change
I had some pain to set the `offset` parameter, so I hope my addition (inspired from other integration docs, e.g. for `scan_interval`), will help others.



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->


## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

Friedrieck committed:
Update gtfs.markdown

To clarify what is expected for the offset parameter (based on the error message I got in the log).

homeassistant commented:
Hi @Friedrieck, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#21670).

Once you do that we will be able to review and accept this pull request.

Thanks!

frenck commented on source/_integrations/gtfs.markdown:
```diff
@@ -62,10 +62,10 @@ name:
   required: false
   default: 0
-  type: [integer, time]
+  type: [integer (seconds), time]
```
"integer (seconds)" is not a valid data type.
Please revert this change

Friedrieck replied:
Done.

Friedrieck committed:
offset type corrected

Description amended to specify that integer value is in seconds (to note: the `offset` attribute of the sensor is in minutes).

frenck reviewed (dismissed):
Thanks, @Friedrieck 👍

frenck commented on source/_integrations/gtfs.markdown:
```diff
@@ -62,7 +62,7 @@ name:
   type: string
 offset:
-  description: A minimum delay to look for. If a departure is in less time than `offset`, it will be ignored.
+  description: A minimum delay to look for. If a departure is in less time than `offset`, it will be ignored. Set the value either in seconds (`integer`), or using one of these `time` formats: `offset: 'HH:MM:SS'`, `offset: 'HH:MM'`. 
```
```suggestion
  description: "A minimum delay to look for. If a departure is in less time than `offset`, it will be ignored. Set the value either in seconds (`integer`), or using one of these `time` formats: `offset: 'HH:MM:SS'`, `offset: 'HH:MM'`. "
```

frenck committed:
Tweak
